strings/base_coroutine_threadpool.h

```diff
@@ -109,14 +109,15 @@ namespace winrt::impl
     template <typename T>
     class awaiter_finder
     {
-        template <typename U, typename = decltype(std::declval<U>().await_ready())> static constexpr bool find_awaitable_member(int) { return true; }
         template <typename> static constexpr bool find_awaitable_member(...) { return false; }
-
-        template <typename U, typename = decltype(std::declval<U>().operator co_await())> static constexpr bool find_co_await_member(int) { return true; }
         template <typename> static constexpr bool find_co_await_member(...) { return false; }
+        template <typename> static constexpr bool find_co_await_free(...) { return false; }
 
+#ifdef WINRT_IMPL_COROUTINES
+        template <typename U, typename = decltype(std::declval<U>().await_ready())> static constexpr bool find_awaitable_member(int) { return true; }
```

Comment:
This one doesn't mention `operator co_await` and could stay in its original location, I think.

Reply:
It was just cleaner to group the trues and falses together.